### DISCUSSION TITLE
Passing cache to workflows on different repository

### DISCUSSION THREAD

**Discussion by ohbus** [github/india#67] [Satellite India 2021 Q&A]
Hi,

Can GitHub Actions pass cache stored _(maven artifacts in my case)_ from one repo to workflows triggered in another repo?

I would like to know if matching the key would result in a cache hit from other repository?

**Comment by msupernaut** [comment]
hey @ohbus - I'd recommend you check out http://support.github.com/ 

There is a specific topic : https://github.community/t/caching-files-between-github-action-executions that might help you get started.

**Reply by ohbus** [reply]
I have already gone down through this community topic but it is not helpful as it suggests to upload and download artifacts which is not useful for workloads as our goal is to reuse the cache from different repositories.

**Comment by FRTMentor** [comment]
Please reach out the GitHub support team.
